bin/elasticurl/main.c

```diff
@@ -515,6 +551,14 @@ int main(int argc, char **argv) {
     app_ctx.output = stdout;
     app_ctx.verb = "GET";
     aws_mutex_init(&app_ctx.mutex);
+    aws_hash_table_init(
+        &app_ctx.signing_context,
+        allocator,
+        10,
```

Comment:
Shouldn't this be 1 or 2? There's only a single kv pair ever stored in here, right?